build/mac/sign_dmg.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [[ $# -lt "5" ]]; then
+  echo "usage: $0 <dmg_path>"
+  exit 1
+fi
+
+SOURCE="${1}"
+DEST="${2}"
+MAC_SIGNING_KEYCHAIN="${3}"
+MAC_SIGNING_IDENTIFIER="${4}"
+REQUIREMENTS="${5}"
+
+app_name="$(basename $SOURCE)"
+
+set -v
+
+function check_exit() {
+    return=$?;
+    if [[ $return -eq 0 ]]; then
+  echo "[INFO] $0 succeded"
+    else
+  echo "[ERROR] $0 failed"
+    fi
+
+    exit $return
+}
+
+trap check_exit EXIT
+
+if [[ -f $DEST ]]; then
```

Comment:
I'm fixing this in something I'm working on now, but just for future reference we should not copy files like this in a python script called from gn. If files need to be copied, it should be done in a gn copy target.